packages/web-pkg/src/components/FilesList/ResourceTable.vue

```diff
@@ -864,7 +864,7 @@ export default defineComponent({
       return this.clipboardResources.some((r) => r.id === resource.id)
     },
     shouldDisplayThumbnails(item: Resource) {
-      return this.areThumbnailsDisplayed && !isResourceTxtFileAlmostEmpty(item)
+      return !isResourceTxtFileAlmostEmpty(item)
```

Comment:
What do you think about dropping this whole function as well? While I get the intention, I still always think that it might be confusing at the same time that sometimes .txt files have a preview/thumbnail and sometimes they don't. Yes, the thumbnail is a white box if the text file is empty. But well, so be it... I'm in favour of reducing complexity / confusion and remove the helper and its usages entirely. Would fit in this PR...

Reply:
Currently idk, it seems that the server is throwing an error, when you request a preview on an empty thumbnail which triggers, the CI to fail.
Should I proceed ?